pkg/syfs/syfs.go

```diff
@@ -61,6 +62,15 @@ func configDir(dir string) string {
 	}
 
 	homedir := os.Getenv("HOME")
+	needUpdate := false
+	if homedir != "" && !fs.IsWritable(homedir) {
```

Comment:
The reported issue is for non-readable, not non-writable.

Reply:
thanks! I have updated the code.